htcosmos/make_parquet.py

```diff
@@ -65,6 +65,46 @@ def tlog(msg):
     now = datetime.now().strftime('%X.%f')
     print(f'{now} {msg}')
 
+def tlog_flush(msg):
```

Comment:
It looks like this is just to add timestamps to print statements -- one approach might be instead to use the standard [logging](https://docs.python.org/3/library/logging.html). I think the main benefits of doing it this way would be the ability to set logging levels and that it should play well with logging from other components (e.g. output from the detection)

Adding (untested)
```
import logging
logging.basicConfig(format='%(levelname)s :: %(asctime)s :: %(message)s', level=logging.DEBUG) # logs log level, time, and message for DEBUG, INFO, WARN, and ERROR logs
logger = logging.getLogger(__name__)
```
and replacing tlog/tlog_flush calls with:

```
logger.DEBUG(message) # or whatever level is appropriate
```

Reply:
other parts of the software stack here appear to be using standard logging.  which would then interleave my logging messages with theirs.   also standard logging seems to write to stderr not stdout,